pubsublite/spark-connector/spark_streaming_to_pubsublite_example.py

```diff
@@ -35,13 +36,32 @@ def spark_streaming_to_pubsublite(
     # |-- value: long (nullable = true)
     sdf = spark.readStream.format("rate").option("rowsPerSecond", 1).load()
 
+    # Transform the dataframe to match the required data fields and data types:
+    # https://github.com/googleapis/java-pubsublite-spark#data-schema
```

Comment:
Is this the canonical link for this schema?

Reply:
Yes. It's the only place it exists for now.